config/routes.rb

```diff
@@ -1,8 +1,10 @@
 Rails.application.routes.draw do
-
-  get 'static_pages#home'
-  get 'static_pages/help'
-  get 'static_pages/about'
-  get 'static_pages/contact'
-  root 'application#hello'
+  scope "(:locale)", locale: /en|vi/ do
+    get "users/new"
+    root "static_pages#home"
+    get  "/help",    to: "static_pages#help"
```

Comment:
sau get cách 1 spaces, sau , cách 1 spaces
```suggestion
    get "/help", to: "static_pages#help"
```